vagen/envs/primitive_skill/benchmark.py

```
@@ -0,0 +1,309 @@
+        f"Benchmark: {num_rounds} rounds x {num_clients} clients x {num_steps} steps "
+        f"[env={env_label}]"
+    )
+    LOGGER.info(f"Server: {base_url}")
+
+    # Warm-up: quick health check
+    import httpx
+    async with httpx.AsyncClient(timeout=10) as hc:
+        try:
+            resp = await hc.get(f"{base_url}/health")
+            LOGGER.info(f"Health check: {resp.json()}")
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)